Aster keep, ancyra, aerinwild, oh my!

SlyViolin opened the pull request:
## Checklist

The list below serves as a reminder to make sure, all needed changes were added to the repository. You can add a `x` between the square brackets to tick the items when in edit mode or just tick the checkboxes when not.

If you forgot something, don't worry. Just do the changes on the same branches and they are added automatically to the pull request.

- [ ] Added changed map file
- [ ] Used a descriptive pull request title
- [ ] Wrote a short description of the change in the pull request description (see below). Include area names where possible.
- [ ] Added a short sentence summarizing the changes to the `changelog.txt` file
- [ ] Increased the number in `version.txt` by 1

## Included changes

SlyViolin committed:
Aster + Bold. + General fixes

- Added Inner Aster Town
- Added Boldarihm (Partial)
- LHG Cleanup ( deleted 76, aligned 11935)
- 4545 Assigned as commodityshop feature
- Partial Wormhole cleanup.

SlyViolin committed:
Update version.txt

SlyViolin committed:
Aster + Bold. + Gen. Fixes

SlyViolin committed:
Update version.txt

keneanung committed:
Merge remote-tracking branch 'upstream/development' into SlyViolinDev

keneanung committed:
Lock wormholes

keneanung committed:
Merge remote-tracking branch 'upstream/development' into SlyViolinDev

SlyViolin committed:
Merge branch 'development' of https://github.com/SlyViolin/AchaeaCrowdmap into development

SlyViolin committed:
Create map.json

SlyViolin committed:
Aster Keep, Ancyra, Wormholes, oh my!

- Added Aster Keep
- Added Ancyra (House)
- Removed destroyed wormholes
- Re-directed broken wormholes
- Quick Hashan shop title update

SlyViolin committed:
Merge branch 'development' of https://github.com/SlyViolin/AchaeaCrowdmap into development

SlyViolin committed:
Aster Keep, Ancyra, Aerinwild, Oh my!

- Added Aster Keep
- Added Ancyra (House)
- Updated Aerinwild
- Removed destroyed wormholes
- Re-directed broken wormholes
- Quick Hashan shop title update

keneanung commented:
Room 55762 now has 2 special exists: `worm warp` and `wormwarp`. The latter one has to be removed or people will get stuck there quite a lot.

Also danger states that there are unlocked wormholes.

SlyViolin committed:
Aster Keep, Ancyra, Aerinwild, Oh my!

- fixed the room 55762 wormwarp/worm warp scuffle
- Ensured wormholes are locked.

keneanung reviewed (approved):
Thanks for contributing these improvements!
